qiita_db/sql_connection.py

```diff
@@ -493,6 +493,24 @@ def add_post_rollback_func(self, func, *args, **kwargs):
 TRNADMIN = Transaction(admin=True)
 
 
+def encapsulated_query(sql, parameters=None):
```

Comment:
Name is fine but I think it would be nice to make it clear that queries are executed as transactions? Up to your discretion. What about `execute_as_transaction`.

Reply:
Yeah, selected `single_query` first, then changed it to `encapsulted_query` but still not happy. `execute_as_transaction` is a decorator we hace; thus, gonna change to `perform_as_transaction`.

Reply:
👍